react/features/toolbox/actions.web.js

```diff
@@ -218,28 +215,22 @@ export function showSharedVideoButton(): Function {
 }
 
 /**
- * Shows SIP call button if it's required and appropriate
- * flag is passed.
+ * Shows SIP call button if it's required and appropriate flag is passed.
  *
  * @param {boolean} show - Flag showing whether to show button or not.
  * @returns {Function}
  */
 export function showSIPCallButton(show: boolean): Function {
-    return (dispatch: Dispatch<*>) => {
+    return (dispatch: Dispatch<*>, getState: Function) => {
         const buttonName = 'sip';
 
-        // hide the button if there is a config to check for user roles,
-        // based on the token and the the user is guest
-        const shouldShow
-            = APP.conference.sipGatewayEnabled()
+        if (show
+                && APP.conference.sipGatewayEnabled()
                 && UIUtil.isButtonEnabled(buttonName)
-                && show
                 && (!config.enableUserRolesBasedOnToken
-                        || !APP.tokenData.isGuest);
-
-        if (shouldShow) {
+                    || !getState()['features/jwt'].isGuest)) {
             dispatch(setToolbarButton(buttonName, {
-                hidden: !shouldShow
+                hidden: false
```

Comment:
This function used to hide the button if `show` was false. Is that change intentional?